openmdao/core/group.py

```diff
@@ -341,6 +344,7 @@ def _configure(self):
             if subsys.matrix_free:
                 self.matrix_free = True
 
+        self._problem_meta['_setup_status'] = 1
```

Comment:
For consistency, problem metadata variables shouldn't begin with underscores, and use a constant instead of 1.